real_time/arduino_src/radio_buggy_mega/main.cpp

```diff
@@ -392,6 +394,15 @@ int main(void)
 
         if(g_is_autonomous)
         {
+
+            // make sure that high-level can't send us obscenely large angles
+            if(auto_steering_angle < STEERING_LIMIT_LEFT) {
```

Comment:
ok but why not do it for teleop and auton?  I just don't see why the validation code for what an allowed angle is should be any different?